db/LMS.sql

```diff
@@ -0,0 +1,16 @@
+DROP DATABASE IF EXISTS lms;
+CREATE DATABASE lms;
+
+\connect lms
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE USERS(
+    id uuid UNIQUE DEFAULT uuid_generate_v4(),
+    email VARCHAR(128) NOT NULL UNIQUE PRIMARY KEY,
+    password VARCHAR(128) NOT NULL,
```

Comment:
I expect you include other info like first name, last name, gender, pronoun date of birth etc.